Assignment1/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,500;0,700;0,900;1,300;1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">  
+    <script src="app.js"></script>
+    <title>Canteen Koala</title>
+</head>
+
+<body>
+    
+    <div id="upper">
```

Comment:
What does upper mean in this context? I assume this is the top-section of the page. 
I would rename this to `top-section`